src/plugins/gui/fiatPlugin.tsx

```diff
@@ -249,6 +249,9 @@ export const executePlugin = async (params: {
     },
     exitScene: async () => {
       navigation.pop()
+    },
+    waitForAnimationFrame: async () => {
+      await new Promise(resolve => requestAnimationFrame(resolve))
```

Comment:
I wonder if calling this `prepareForBigCalculation` would make more sense, since it's goal is to get things in shape before clogging the CPU.